plugins/copr.py

```diff
@@ -254,6 +269,15 @@ def _remove_repo(cls, repo_filename):
             raise dnf.exceptions.Error(str(e))
 
     @classmethod
+    def _disable_repo(cls, copr_username, copr_projectname):
+        exit_code = call(["dnf", "config-manager", "--set-disabled",
```

Comment:
We cannot find any better workaround to call plugins from plugins than external `call`. The code in Michaels comment will not work if the called plugin requests different demands. So I am fine with this line until we find better solution. @xsuchy , you can merge it once the issue with search command is fixed.